src/main/java/mate/academy/springboot/datajpa/dto/ProductRequestDto.java

```diff
@@ -0,0 +1,15 @@
+package mate.academy.springboot.datajpa.dto;
+
+import java.math.BigDecimal;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class ProductRequestDto {
+    private String title;
+
```

Comment:
Not fixed after two mentors' comments.
Delete this empty space

Reply:
took away